### PR TITLE
DLS-6253: refactoring and import optimisation

### PR DESCRIPTION
**Pull request by johno1985**
some minor refactoring and tidy up
import optimisations

### CODE REVIEW

File: app/uk/gov/hmrc/cbcrfrontend/model/DocRefIdPair.scala [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ */
+
+package uk.gov.hmrc.cbcrfrontend.model
+
+import play.api.libs.json._
+
+case class DocRefIdPair(docRefId: DocRefId, corrDocRefId: Option[CorrDocRefId])
+object DocRefIdPair {
+  implicit val format: OFormat[DocRefIdPair] = Json.format[DocRefIdPair]
+}
```
Comment (LAJW) [inline app/uk/gov/hmrc/cbcrfrontend/model/DocRefIdPair.scala:24]: I'm not sure it is an improvement to put these in their own files.

File: app/uk/gov/hmrc/cbcrfrontend/model/PartialReportingEntityData.scala [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cbcrfrontend.model
+
+import play.api.libs.json._
+
+import java.time.LocalDate
+
+case class PartialReportingEntityData(
```
Comment (LAJW) [inline app/uk/gov/hmrc/cbcrfrontend/model/PartialReportingEntityData.scala:23]: Having this in its own file makes it more difficult to differentiate between it and `ReportingEntityData`.

File: app/assets/javascripts/timeout-dialog.js [hunk 3]
```diff
@@ -24,68 +24,75 @@
  * THE SOFTWARE.
  */
 
-Date.now = Date.now || function() { return +new Date; };
- if (!window.console){ console = {log: function() {}} };
+Date.now = Date.now || function () {
```
Comment (LAJW) [inline app/assets/javascripts/timeout-dialog.js:27]: This commit seems mislabeled - it says "remove assets", but here the assets seem reformatted?

**Review by LAJW — COMMENTED** [review]
I do like a "domain types" files which contain the case classes used in the domain model. Splitting them seems unnecessary, the file could have a better name, but I don't think having a 2-field methodless record in its own file is an improvement - makes it harder to understand the model in my opinion.

Also, the assets folder changes don't seem labelled correctly - I think it was reformatted, not removed?

**Review by LAJW — COMMENTED** [review]
The title seems misleading as well - there aren't any changes pertaining to build.sbt or scala versions.

File: app/assets/javascripts/timeout-dialog.js (hunk 3 above)
Comment (johno1985) [reply app/assets/javascripts/timeout-dialog.js:27]: Strange, I removed them, will fix and re-commit

File: app/uk/gov/hmrc/cbcrfrontend/model/DocRefIdPair.scala (hunk 1 above)
Comment (johno1985) [reply app/uk/gov/hmrc/cbcrfrontend/model/DocRefIdPair.scala]: Have moved this into PartialReportingEntityData

**Comment by johno1985** [comment]
Have updated PR title and added some comments

**Pull request merged**